instance/tests/models/test_openedx_instance.py

```diff
@@ -467,6 +467,10 @@ def test_get_load_balancer_configuration(self, mocks):
             backend_map, config, domain_names, instance.active_appserver.server.public_ip
         )
 
+        # Test configuration after terminating appserver
+        with patch('instance.openstack.get_server_public_address', return_value=None):
+            self.assertEqual(instance.get_load_balancer_configuration(), ([], []))
```

Comment:
@smarnach No, you're right, it's not worth it here. Maybe describe the bug we had though, with a ticket/comment URL, so we don't end up removing inadvertently, and can easily get the context if needed?